app/uk/gov/hmrc/gform/models/FieldValueMaker.scala

```diff
@@ -60,22 +67,28 @@ class FieldValueMaker(json: JsValue) {
   lazy val infoText: Option[String] = (json \ "infoText").asOpt[String]
   lazy val infoType: Option[String] = (json \ "infoType").asOpt[String]
   lazy val shortName: Option[String] = (json \ "shortName").asOpt[String]
-  lazy val repeatsMax: JsResult[Option[Int]] = (json \ "repeatsMax").validateOpt[Int]
-  lazy val repeatsMin: Option[Int] = (json \ "repeatsMin").asOpt[Int]
+  lazy val jsResultMaybeRepeatsMax: JsResult[Option[Int]] = (json \ "repeatsMax").validateOpt[Int]
+  lazy val jsResultMaybeRepeatsMin: JsResult[Option[Int]] = (json \ "repeatsMin").validateOpt[Int]
```

Comment:
Here we could return `Opt[...]` instead of `JsResult` because later is not operable in for comprehension toghether with `Either`. Other solution is to convert all other fields to be `JsResult` rather than `Either`.